### PR TITLE
feat(website): add cloudbuild config for auto-updating website

### PR DESCRIPTION
**Pull request by justbuchanan**
See Cloud Build "trigger" here: https://console.cloud.google.com/cloud-build/triggers;region=global/edit/24ce2421-6f4e-4d6c-a715-217d714cbe31?project=kythe-repo

And a recent successful build/deploy here: https://console.cloud.google.com/cloud-build/builds;region=global/699028f6-4cec-4cc5-a6b8-76fd8dc662a9?project=kythe-repo

### CODE REVIEW

File: kythe/release/cloudbuild/website-updater/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/kythe-repo/bazelisk-builder-client
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y \
```
Comment (shahms) [inline kythe/release/cloudbuild/website-updater/Dockerfile:18]: These should be a single docker `RUN` directive: https://docs.docker.com/develop/develop-images/instructions/#run

File: kythe/release/cloudbuild/website-updater/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    asciidoc asciidoctor source-highlight graphviz ruby-dev build-essential \
+    rsync jq
+
+# install `gcloud` cli tool
+RUN curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-466.0.0-linux-x86_64.tar.gz
+RUN tar -xf google-cloud-cli-466.0.0-linux-x86_64.tar.gz
+RUN ./google-cloud-sdk/install.sh
```
Comment (shahms) [inline kythe/release/cloudbuild/website-updater/Dockerfile:25]: These should *likely* be a single directive

File: kythe/release/cloudbuild/website-updater/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ENV PATH="${PATH}:/google-cloud-sdk/bin"
+RUN which gcloud
```
Comment (shahms) [inline kythe/release/cloudbuild/website-updater/Dockerfile:27]: what's the purpose of the call to `which`?

File: kythe/release/cloudbuild/website-updater/Dockerfile (hunk 1 above)
Comment (justbuchanan) [reply kythe/release/cloudbuild/website-updater/Dockerfile]: Done.

File: kythe/release/cloudbuild/website-updater/Dockerfile (hunk 3 above)
Comment (justbuchanan) [reply kythe/release/cloudbuild/website-updater/Dockerfile]: Removed - I initially had it to make sure the `ENV` above it worked.

**Review by shahms — APPROVED** [review]

**Pull request merged**